src/libsyntax/feature_gate.rs

```diff
@@ -230,6 +230,13 @@ declare_features! (
     // Allows panicking during const eval (produces compile-time errors)
     (active, const_panic, "1.30.0", Some(51999), None),
 
+    // Don't run any static analyses on constants and `const fn` and just "let miri handle it"
+    // NOT meant to ever be stabilized, this is solely for being able to demonstrate problems with
+    // features of alternative RFCs when writing RFCs that would allow more const features.
+    //
+    // rustc internal
+    (active, unleash_the_miri_inside_of_you, "1.31.0", None, None),
```

Comment:
Shouldn't it have a prefix `rustc_` if it is internal?

Reply:
aaaw :(

can we do a postfix and make it `unleash_the_miri_inside_of_rustc`? :laughing: 

The feature gate name can be bikeshed of course, but it should be riddiculous enough to be self-documenting about its "don't use this except for demo purposes"-nature. It is literally unsound to use this feature gate.

Reply:
It should probably also contain "unsound" or "unsafe", then...

Reply:
I don't care so much; it's just customary to have the `rustc` prefix is all =P
If you want to make it postfix + include `unsound` then pew pew ;)